security/pkg/server/ca/authenticate/kube_jwt_test.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 Istio Authors
+// Copyright 2019 Istio Authors
```

Comment:
Add tests on using trustworthy-jwt and legacy-jwt?

Reply:
We have tests here for requests with both legacy jwts and trustworthy jwts.
https://github.com/istio/istio/pull/15568/files#diff-8347ee5088791c0b1027d5c6b8a595e4R62. I think it might be redundant to add tests here since `KubeJWTAuthenticator` just calls the same function, i.e. `ValidateK8sJwt()`.

Reply:
Got it, thanks.